test/components/countdown-test.js

```diff
@@ -165,7 +165,7 @@ describe('Countdown tests', function() {
       });
 
       it('should return 10 days', function() {
-        assert.equal(duration.asDays(), -10);
+        assert.equal(Math.ceil(duration.asDays()), -10);
```

Comment:
this is not the right fix for this imo, if it started returning 9.5 or 9.00001 we'd not know... we just need to cover the clocks going back, which you can do with this:
```
   describe('_durationFromNow', function() {
     var clock;
    beforeEach( function( ) {
      clock = sinon.useFakeTimers( new Date( 2015, 1, 1 ).getTime( ));
    } );
    afterEach( function( ) {
      return clock.restore( );
    } );
```